src/player.js

```diff
@@ -636,6 +636,33 @@ export default class Player extends FakeEventTarget {
     this.dispatchEvent(new FakeEvent(CustomEventType.PLAYER_DESTROY));
     this._eventManager.destroy();
   }
+  /**
+   * Listen to the video element events and triggers them from the engine.
+   * @private
+   * @param {FakeEvent} event data
+   * @param {?boolean} stopSeeking don't seek to the last detach point
+   * @returns {void}
+   */
+  _attachMediaSource(event: FakeEvent, stopSeeking: ?boolean): void {
```

Comment:
`event` is useless

Reply:
but require because it's handler and it's his default

Reply:
@Yuvalke what do you mean? just remove the passing of it form the event registration:
`this._eventManager.listen(this, AdEventType.ALL_ADS_COMPLETED, e => this._attachMediaSource(e, true));`
`this._eventManager.listen(this, AdEventType.ALL_ADS_COMPLETED, () => this._attachMediaSource(true));`